src/docker-2ndboot.sh

```diff
@@ -44,6 +44,11 @@ if ! systemctl is-system-running --wait; then
 fi
 systemctl status --no-pager -l anbox-container-manager
 
+until [ -e /run/anbox-container.socket ]; do
+    echo "Waiting for ready /run/anbox-container.socket"
```

Comment:
Should exit after retrying for a few minutes

Reply:
Timeout applied to all waits. review please. thank you!